src/suggestion.rs

```diff
@@ -315,6 +315,24 @@ pub struct Suggestion<'s> {
     pub description: Option<String>,
 }
 
+impl<'s> Suggestion<'s> {
+    /// Determine if there is overlap.
+    pub fn is_overlapped(&self, other: &Self) -> bool {
+        if self.origin != other.origin {
+            return false;
+        }
+        if self.span.start.line != other.span.start.line {
+            return false;
+        }
```

Comment:
The first line being equal is not necessarily a good indicator, right now this _might_ work, but the plan is to have multiline corrections eventually.